### ISSUE TITLE
SDK 3.1.415: dotnet doesn't work on MacOs 10.13 (High Sierra)

### ISSUE THREAD

**Issue by borjafdezgauna** [dotnet/sdk#22782]
Hi, Not sure this is the right place to report this possible issue or if the assumed bug is intended behavior. Please forgive me if it's not

I installed the latest release of the SDK on MacOs High Sierra, tried to run "dotnet" and got an error complaining about a library (libSystem.B...) that was compiled for MacOs 11.0. According to the [compatibility chart](https://github.com/dotnet/core/blob/main/release-notes/3.1/3.1-supported-os.md) NetCore3.1 should be compatible with MacOs 10.13, right?

I uninstalled it, installed SDK 3.1.413 and it worked fine.

This suspected incompatibilty made the artifacts produced by my Azure pipeline stop being compatible with MacOs High Sierra/Mojave/Catalina, even if they were being compiled using NetCore3.1

Cheers!

**Comment by ryank425** [comment]
I think 10.13+ on documentation is wrong since High Sierra (10.13) and Mojave (10.14) have reached end of life from Apple long time ago https://endoflife.date/macos and they are not receiving any support from Apple. It should have been 10.15 (Catalina) +

**Comment by marcpopMSFT** [comment]
@rbhanda to follow up on the support question.

**Comment by rbhanda** [comment]
https://github.com/dotnet/core/pull/6984 To update the macOS supported versions in release notes.

**Comment by kubakista** [comment]
It is kind of strange to just update user docs if 3.1.413 is working fine, and suddenly 414 and 415 are not working anymore...

**Comment by KalleOlaviNiemitalo** [comment]
Yes, please record the end of support in all the places listed at [Change Process](https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md#change-process).
